rack-box/scripts/install.sh

```diff
@@ -90,6 +91,8 @@ systemctl start fuseki
 cd /home/"${USER}"/RACK/rack-ui
 python3 -m pip install -r ./requirements.txt
 adduser --system --group --no-create-home --disabled-password rackui
+usermod -aG sudo rackui
+echo "rackui ALL=(ALL) NOPASSWD:ALL" | tee -a /etc/sudoers
```

Comment:
Likewise, I suggest changing line 95 to create a different and new file to reduce conflict with user-data:

`echo 'rackui ALL=(ALL) NOPASSWD:ALL' > /etc/sudoers.d/rackui`

That way, both user-data and install.sh will end up creating the same file with the same contents anyway.